README.md

```diff
@@ -155,7 +157,9 @@ Pre-compiled binaries are available for released versions in the [releases secti
 
 ### Building from source
 
-For more information on how to build from source please refer to the [Svix server README](./server/).
+The Svix server is written in Rust 🦀 and requires a Rust build environment.
+
+If you already have one, you just need to run `cargo build`, otherwise, please please refer to the [Svix server README](./server/) for more information about building the server from source.
```

Comment:
```suggestion
If you already have one, you just need to run `cargo build`, otherwise, please please refer to the [Svix server README](./server/#readme) for more information about building the server from source.
```